.github/ISSUE_TEMPLATE/library-bug.yml

```diff
@@ -0,0 +1,36 @@
+name: Library Bug Report (developers)
+description: File a bug report on one of our NPM libraries
+title: "[Bug]: "
+labels: ["bug", "libraries", "Needs Triage"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+  - type: input
+    id: library
+    attributes:
+      label: Impacted Library name
+      description: What is the name of the NPM library you used?
```

Comment:
It would be great to ask about the version of the library which is impacted by the bug